src/components/AppProvider/utilities/createAppProviderContext/createAppProviderContext.ts

```diff
@@ -14,6 +14,25 @@ export interface CreateAppProviderContext extends AppProviderProps {
   unsubscribe?(callback: () => void): void;
 }
 
+const serverAppBridge = {
```

Comment:
I'm thinking that the `noop` version of App Bridge should live inside the App Bridge repo as Polaris shouldn't care about the implementation details. However, for a quick fix we can just add it to Polaris first.

@tmlayton @lemonmade Thoughts?